bin/tessel-2.js

```diff
@@ -38,6 +38,20 @@ function closeFailedCommand(err) {
   process.exit(1);
 }
 
+function makeCommand(commandName) {
+  return parser.command(commandName)
+    .option('timeout', timeoutOption)
+    .option('name', nameOption)
+    .option('lan', {
```

Comment:
I think we should either break the `lan` and `usb` option into separate variables just like `timeoutOption` and `nameOption` are or bring the `timeoutOption` and `nameOption` back into this function (for consistency). I don't think those two vars are used anywhere else.